apps/opik-backend/src/main/java/com/comet/opik/domain/DashboardService.java

```diff
@@ -0,0 +1,386 @@
+import jakarta.inject.Inject;
+import jakarta.inject.Provider;
+import jakarta.inject.Singleton;
+import jakarta.ws.rs.NotFoundException;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.jdbi.v3.core.statement.UnableToExecuteStatementException;
+import ru.vyarus.guicey.jdbi3.tx.TransactionTemplate;
+
+import java.sql.SQLIntegrityConstraintViolationException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import java.util.UUID;
+
+import static com.comet.opik.infrastructure.db.TransactionTemplateAsync.READ_ONLY;
+import static com.comet.opik.infrastructure.db.TransactionTemplateAsync.WRITE;
+
+@ImplementedBy(DashboardServiceImpl.class)
+public interface DashboardService {
+
+    Dashboard create(Dashboard dashboard);
+
+    Dashboard update(UUID id, DashboardUpdate dashboardUpdate);
+
+    void delete(UUID id);
+
+    void delete(Set<UUID> ids);
+
+    Dashboard findById(UUID id);
+
+    Optional<Dashboard> findByName(String name);
+
+    Dashboard.DashboardPage find(int page, int size);
+
+    void associateExperimentWithDashboard(UUID experimentId, UUID dashboardId);
+
+    void removeExperimentDashboardAssociation(UUID experimentId);
+
+    Optional<ExperimentDashboard> findExperimentDashboard(UUID experimentId);
+
+    // Section and Panel creation methods
+    DashboardSection createSection(UUID dashboardId, String title, Integer positionOrder);
+
+    DashboardPanel createPanel(UUID dashboardId, UUID sectionId, String name, String type, Object configuration,
+            Object layout);
+}
+
+@Singleton
+@RequiredArgsConstructor(onConstructor_ = @Inject)
+@Slf4j
+class DashboardServiceImpl implements DashboardService {
+
+    private static final String DASHBOARD_ALREADY_EXISTS = "Dashboard already exists";
+
+    private final @NonNull IdGenerator idGenerator;
+    private final @NonNull TransactionTemplate template;
+    private final @NonNull Provider<RequestContext> requestContext;
+
+    @Override
+    public Dashboard create(@NonNull Dashboard dashboard) {
+        String workspaceId = requestContext.get().getWorkspaceId();
+        String userName = requestContext.get().getUserName();
+
+        UUID id = idGenerator.generateId();
+
+        try {
+            return template.inTransaction(WRITE, handle -> {
+                var dao = handle.attach(DashboardDAO.class);
+
+                // Insert dashboard
+                dao.insert(id, dashboard.name(), dashboard.description(), workspaceId, userName);
+
+                // Insert sections and panels if provided, or create a default section
+                List<DashboardSection> sectionsToCreate = dashboard.sections();
+                if (sectionsToCreate == null || sectionsToCreate.isEmpty()) {
+                    // Create a default section
+                    UUID defaultSectionId = idGenerator.generateId();
```

Comment:
Where is the default section logic found? I couldn't find it anywhere. I expect the BE to be able to respond to a request like `getDefaultDashboard`